### ISSUE TITLE
Commit 1f43e4d breaks the hyperplonk compilation

### ISSUE THREAD

**Issue by alxkzmn** [privacy-scaling-explorations/chiquito#279]
https://github.com/privacy-scaling-explorations/chiquito/commit/1f43e4df4d0be77e49b6cb7c5d62dd4ab04cbb71

Steps to reproduce:
```
cargo run --example fibonacci  -- --nocapture
```
```
thread 'main' panicked at ~/.cargo/git/checkouts/plonkish-69cbf4324857e2fa/e0125fe/plonkish_backend/src/poly/multilinear.rs:52:13:
assertion `left == right` failed
  left: 17
 right: 16
```
Previous commit (6bbe1fe) is still working fine.

**Comment by leolara** [comment]
Fix if it is easy to fix :-) but not a priority now

**Comment by alxkzmn** [comment]
Fixed in #286
